_posts/2016-03-07-2016-forman-survey-analysis.md

```diff
@@ -0,0 +1,342 @@
+Katello development were common.
+
+## <a name="events"></a>Events
+
+We split this up this year. Last year we only asked if you would like to attend
+events - this time we also asked if you'd be willing to help organise them:
+
+![Meetups](/static/images/blog_images/2016-03-07-2016-forman-survey-analysis/meetups.png)
+
+We see that the US and EMEA are popular, but a sizeable chunk of people would
+like a local meetup. We're investigating getting involved with other local
+events such as DevOpsDays and so forth - if there's an event near you that you
+think we should be at, [let us know][contact]!
+
+I'm *really* excited that 22 peaople said they'd be willing to speak about
+Foreman at conferences, and 16 people would be willing to help set up events -
+*thanks*! However, this is an anonymous survey! Please, please *email me* with
+me about this - I'm starting to build a collection of potential speakers (and
+their areas of travel), and potential event locations to see what we can match
+up.
+
+Again, if you responded to this question - *email me*! Thanks!
+
+## <a name="contributor-survey"></a>Contributor survey
+
+I'll keep this section brief, but here's two interesting graphs:
+
+![Contribute](/static/images/blog_images/2016-03-07-2016-forman-survey-analysis/do-you-contribute.png)
+
+Clearly, the responses to this question will be strongly skewed in the
+contributor survey. However, I think it's notable that 20% of people felt
+strongly enough to fill this out even though they are *not* currently a
+contributor - it implies there's probably more people in this position. We owe
+it to ourselves to support new contributors and help them get started.
+
+I do have some ideas here, such as more screencasts (how about a "Let's Play"
+of writing a plugin?) and also perhaps a "where can I help" workflow on the
+website, similar to the [Mozilla site](https://www.mozilla.org/en-US/contribute).
+
+![Dev Support](/static/images/blog_images/2016-03-07-2016-forman-survey-analysis/dev-support.png)
+
+Leading on from that, it seems that while we are consistently good at helping
+out once we get to reporting and monitoring PRs, the picture for helping those
+who are stuck (either stuck *in* some code, or stuck just getting started) is a
+bit more mixed. Work is underway to write a [developer handbook][handbook]
+which is helping to clarify this, and further work remains to be done on it.
+This should probably re-emphasize the IRC dev channel as the primary means of
+support when contributing.
+
+## <a name="final-thoughts"></a>Final thoughts
+
+It's fascinating reading through all the comments and stats - there's more data
+in here than I can show without this becoming a book, but I'll be using it
+myself in my role to assist the community further.
+
+Hopefully the above gives you a flavour of where we're at. For myself, I feel
+that we get good marks for our support - essential for such a complex piece of
```

Comment:
I try to make it less complex, but reinforcing that perception isn't helping :smile: